cmd/util/ledger/migrations/deduplicate_contract_names_migration.go

```diff
@@ -66,8 +66,12 @@ func (d *DeduplicateContractNamesMigration) MigrateAccount(
 		return payloads, nil
 	}
 
+	value := contractNamesPayload.Value()
+	if len(value) == 0 {
+		return payloads, nil
```

Comment:
nit: maybe remove empty payload here.

```suggestion
                copy(payloads[contractNamesPayloadIndex:], payloads[contractNamesPayloadIndex+1:])
                payloads = payloads[:len(payloads)-1]
                return payloads
```